api/v1/endpoints/drone.py

```diff
@@ -6,11 +6,13 @@
 
 @router.get('/add_drone_info')
 def add_dron_status(
+    user: str,
     drone_id: int,
     battery: int,
     d_latitude: float,
     d_longitude: float
 ):
-    db.add_dron_status(drone_id, battery, d_latitude, d_longitude)
-    db.update_task_info(d_latitude, d_longitude)
-    return str('Done')
+    ds_result = db.add_dron_status(
+        user, drone_id, battery, d_latitude, d_longitude)
+    ut_result = db.update_task_info(d_latitude, d_longitude)
+    return ds_result, ut_result
```

Comment:
As a result, you should return JSON schema - one of your schemas/* class

hint: [FastAPI Response Model](https://fastapi.tiangolo.com/tutorial/response-model/)